app/Http/Controllers/Airlines/UpdateAirlineController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers\Airlines;
+
+use App\Http\Requests\StoreAirlineRequest;
+use App\Models\Airline;
+use App\Transformers\AirlineTransformer;
+use Flugg\Responder\Contracts\Responder;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Http\JsonResponse;
+
+class UpdateAirlineController
+{
+    public function __invoke(StoreAirlineRequest $request, int $id, Responder $responder): JsonResponse
+    {
+        try {
+            $airline = Airline::findOrFail($id);
+            $airline->name = $request->string('name')->toString();
+            $airline->description = $request->string('description')->toString();
+            $airline->save();
+            return $responder->success($airline, new AirlineTransformer())->respond();
+        } catch (ModelNotFoundException) {
+            return $responder->error(JsonResponse::HTTP_NOT_FOUND, 'Airline not found.')->respond();
+        }
+    }
```

Comment:
```suggestion
    public function __invoke(StoreAirlineRequest $request, Airline $airline, Responder $responder): JsonResponse
    {
        $airline->update($request->validated());
        
        return $responder
            ->success($airline, AirlineTransformer::class)
            ->respond();
    }
```
With Route Model Binding, your controller will look like this. Also, please don't use try-catch in controllers. It's not a controller's concern to handle what should happen if an exception occurs. That's the exception handler's concern. You may read more about error handling [here](https://laravel.com/docs/11.x/errors).